CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/AutosaveGraphPlugin.java

```diff
@@ -56,8 +56,8 @@
  * @author algol
  */
 @ServiceProvider(service = Plugin.class)
-@PluginInfo(pluginType = PluginType.NONE, tags = {"LOW LEVEL"})
 @Messages("AutosaveGraphPlugin=Autosave Graph")
+@PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
```

Comment:
I was thinking `EXPORT` for the auto save as its creating a file outside of Constellation.

Reply:
Agree

Reply:
#1259